### DISCUSSION TITLE
windows必备软件

### DISCUSSION THREAD

**Discussion by giscus** [oldwinter/knowledge-garden#12] [备发布站的评论系统]
# windows必备软件

[[everything]]

https://notes.oldwinter.top/windows%E5%BF%85%E5%A4%87%E8%BD%AF%E4%BB%B6

**Comment by neverup7** [comment]
补充2个：
potplayer
Bandizip

**Reply by oldwinter** [reply]
哈哈，感谢建议。有机会试试。我换公司后1年多没用win了，以前的软件使用记录直接拖进来也有点落后了，所以在这里挖了坑想将来有机会填，没想到你帮忙一起填坑了。

**Reply by aswirl4** [reply]
banzip除了支持代码页以外我想不到优点了 为什么不用7zip

**Reply by neverup7** [reply]
1.windows11右键Bandizip体验比7zip好
2.UI好一点(个人喜好了)

**Comment by oldwinter** [comment]
试一下邮件回复会发生什么。

在 2022年8月27日星期六，我永不服输！ ***@***.***> 写道：

**Comment by neverup7** [comment]
补充一个浏览器插件推荐
详情:https://github.com/neverp/chajian




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月27日(星期六) 下午5:26
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [oldwinter/knowledge-garden] windows必备软件 (Discussion #12)





 试一下邮件回复会发生什么。 
 
 在 2022年8月27日星期六，我永不服输！ ***@***.***&gt; 写道： 
 
 &gt; 1.windows11右键Bandizip体验比7zip好 
 &gt; 2.UI好一点(个人喜好了) 
 &gt; 
 &gt; — 
 &gt; Reply to this email directly, view it on GitHub 
 &gt; <" rel="noopener" target="_blank">https://github.com/oldwinter/knowledge-garden/discussions/12#discussioncomment-3488105&gt;;, 
 &gt; or unsubscribe 
 &gt; <" rel="noopener" target="_blank">https://github.com/notifications/unsubscribe-auth/AB54QH2A3QNWT55HSI6NMK3V3GZBXANCNFSM57YULG7A&gt;; 
 &gt; . 
 &gt; You are receiving this because you commented.Message ID: 
 &gt; ***@***.*** 
 &gt; com&gt; 
 &gt;

**Reply by neverup7** [reply]
不错
